File: docs/tutorial/hidden-tests/q3.py
test = {
	"name": "q3",
	"points": 1,
	"hidden": False,
	"suites": [ 
		{
			"cases": [ 
				{
					"code": r"""
					>>> len(data["flavor"].unique()) == 4
					True
					""",
					"hidden": False,
					"locked": False,
				}, 
				{
					"code": r"""
					>>> for l in ["chocolate", "vanilla", "strawberry", "mint"]:
					... 	assert l in data["flavor"].unique()
					""",
					"hidden": False,
					"locked": False,
				}, 
				{
					"code": r"""
					>>> assert type(price_by_flavor) == pd.Series
					""",
					"hidden": False,
					"locked": False,
				}, 
				{
					"code": r"""
					>>> assert len(price_by_flavor) == 4
					""",
					"hidden": False,
					"locked": False,
				}, 
			],
			"scored": False,
			"setup": "",
			"teardown": "",
			"type": "doctest"
		}
	]
}

File: test/test-assign/r-correct/autograder/tests/q3.R
test = list(
  name = "q3",
  cases = list(
    ottr::TestCase$new(
      hidden = FALSE,
      name = NA,
      points = 0.6666666666666666,
      code = {
        testthat::expect_equal(nine, 9)
      }
    ),
    ottr::TestCase$new(
      hidden = FALSE,
      name = NA,
      points = 0.6666666666666666,
      code = {
        testthat::expect_equal(square(16), 256)
      }
    ),
    ottr::TestCase$new(
      hidden = FALSE,
      name = NA,
      points = 0.6666666666666666,
      code = {
        testthat::expect_equal(square(1), 1)
      }
    )
  )
)